optimism/src/mips/witness.rs

```diff
@@ -807,6 +811,49 @@ impl<Fp: Field> Env<Fp> {
         None
     }
 
+    fn snapshot_state_at(&mut self, at: &StepFrequency) {
+        if self.should_trigger_at(at) {
+            let filename = format!("snapshot-state-{}.json", self.instruction_counter);
```

Comment:
Minor quip: shouldn't the format follow the `snapshot-fmt` CLI argument?

Reply:
Good idea, but I couldn't see how it was used. It seems it isn't used, right?